Feature

lusekayz opened the pull request:
My first pull request

mateo951 commented on .gitignore:
```diff
@@ -0,0 +1,3 @@
+# .gitignore
+node_modules/
+test.md
```
Nicely done! Both the folder and the file are ignored in your repo 💪

mateo951 commented on .github/workflows/linters.yml:
```diff
@@ -0,0 +1,48 @@
+name: Linters
```
Please be sure that all of your linters are passing before asking for your next review 😉 Stylelint checks are failing 👀

mateo951 reviewed (changes requested):
### Hi @lusekayz , 

📝👌 **Your project highlights** 🔥😎 
- Nice README file 👀
- Proper use of GitHub Flow 🤯

You have done a very good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)_

mateo951 commented:
Kindly make your Pull Request title a bit more descriptive by telling the main changes/features you are merging into the main branch 😎

Also, please add a short summary describing introduced changes in more detail.
![image](https://user-images.githubusercontent.com/15807343/153033165-11f8323c-58c9-40db-bf50-2c21275993c8.png)

For example, you could go with something like this🤓
  - Added setup for linters
  - Added gitignore file with ignored files/directories 
  - Added main structure (index.html / style.css)

lusekayz commented:
For my first pull request I performed the following tasks.

    Added setup for linters.
    Added gitignore file with ignored files/directories
    Added main structure (index.html / style.css)

The above features were merged to the main feature branch.

ShoiraTa commented:
### **STATUS: APPROVED  ✔️** 
------
Hi @lusekayz ,  

Your project is complete! Congratulations!  🎉💯🌟 

## Good Points 👍 
- Project is set up correctly ✔️ 
- No linter errors ✔️ 
- Professional documentation ✔️ 

<img src="https://www.serialminds.com/wp-content/uploads/2015/11/loser.gif" width="400" height="250" />

Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 
### OPTIONAL

- [ ] Did you know that in GitHub you can easily edit pull requests? To do that you should click on three dots on the right side of pull request 😉 :

![image](https://user-images.githubusercontent.com/77038610/153134259-0fabc130-c68b-4196-a0ce-f8d2a6426012.png)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
